frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/GitNewCommitMessageActionDialog.kt

```diff
@@ -0,0 +1,62 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.openapi.editor.impl.EditorImpl
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.util.Disposer
+import com.intellij.openapi.vcs.ui.CommitMessage
+import com.intellij.ui.components.JBLabel
+import com.intellij.util.ui.JBUI
+import com.intellij.util.ui.UIUtil
+
+internal class GitNewCommitMessageActionDialog(
```

Comment:
Huh as usual... is it based on some existing class or purely your work? If the former, pls _cite your sources_ in a code comment